src/blk/kernel/KernelDevice.cc

```diff
@@ -72,7 +72,7 @@ KernelDevice::KernelDevice(CephContext* cct, aio_callback_t cb, void *cbpriv, ai
   fd_directs.resize(WRITE_LIFE_MAX, -1);
   fd_buffereds.resize(WRITE_LIFE_MAX, -1);
 
-  bool use_ioring = cct->_conf.get_val<bool>("bdev_ioring");
+  use_ioring = cct->_conf.get_val<bool>("bdev_ioring");
   unsigned int iodepth = cct->_conf->bdev_aio_max_queue_depth;
 
   if (use_ioring && ioring_queue_t::supported()) {
```

Comment:
In "else" for this condition
https://github.com/ceph/ceph/pull/60791/files#diff-1b8b5867aeaaa9c7a63ec9488d924befac51d3cbd8ca31806e97ab20a33c8cd7R83-R89
we might still have (use_ioring == true).
Lets `use_ioring=false;` there.